config/OPM-Reconfig.cfg

```
@@ -0,0 +1,391 @@
+		}
+	}
+	
+ 	ASTEROIDGROUP
+	{
+		name = joolOuter
+		title = #autoLOC_CustomAsteroids_OPM_GroupOuterObj
+
+ 		centralBody = Jool
+
+ 		spawnRate = 0.10
+
+ 		orbitSize
+		{
+			dist = Uniform
+			min = Ratio(Tylo.sma, 2)
+			max = Ratio(Pol.sma, 1)
+		}
+
+ 		eccentricity
+		{
+			dist = Uniform
+			avg = 0.2
+			stddev = 0.5
+		}
+
+ 		inclination
+		{
+			dist = Isotropic
+		}
+
+ 		asteroidTypes
+		{
+			key = 0.75 PotatoRoid
+			key = 0.20 CaAsteroidCarbon
+			key = 0.05 CaAsteroidMetal
+		}
+	}
+
+ 	ASTEROIDGROUP
+	{
+		name = sarnusOuter
+		title = #autoLOC_CustomAsteroids_OPM_GroupOuterObj
+
+ 		centralBody = Sarnus
+
+ 		spawnRate = 0.10
+
+ 		orbitSize
+		{
+			dist = Uniform
+			min = Ratio(Slate.sma, 2)
+			max = Ratio(Tekto.sma, 2)
+		}
+
+ 		eccentricity
+		{
+			dist = Uniform
+			avg = 0.2
+			stddev = 0.5
+		}
+
+ 		inclination
+		{
+			dist = Isotropic
+		}
+
+ 		asteroidTypes
+		{
+			key = 0.75 PotatoRoid
+			key = 0.20 CaAsteroidCarbon
+			key = 0.05 CaAsteroidMetal
+		}
+	}
+
+ 	ASTEROIDGROUP
+	{
+		name = urlumOuter
+		title = #autoLOC_CustomAsteroids_OPM_GroupOuterObj
+
+ 		centralBody = Urlum
+
+ 		spawnRate = 0.10
+
+ 		orbitSize
+		{
+			dist = Uniform
+			min = Ratio(Priax.sma, 2)
+			max = Ratio(Wal.sma, 2)
+		}
+
+ 		eccentricity
+		{
+			dist = Uniform
+			avg = 0.2
+			stddev = 0.5
+		}
+
+ 		inclination
+		{
+			dist = Isotropic
+		}
+
+ 		asteroidTypes
+		{
+			key = 0.10 CaAsteroidCarbon
+			key = 0.90 CaAsteroidIcy
+		}
+	}
+
+ 	ASTEROIDGROUP
+	{
+		name = neidonOuter
+		title = #autoLOC_CustomAsteroids_OPM_GroupOuterObj
+
+ 		centralBody = Neidon
+
+ 		spawnRate = 0.15
+
+ 		orbitSize
+		{
+			dist = Uniform
+			min = Ratio(Thatmo.sma, 2)
+			max = Ratio(Nissee.sma, 1)
+		}
+
+ 		eccentricity
+		{
+			dist = Uniform
+			avg = 0.2
+			stddev = 0.5
+		}
+
+ 		inclination
+		{
+			dist = Isotropic
+		}
+		
+ 		asteroidTypes
+		{
+			key = 0.10 CaAsteroidCarbon
+			key = 0.90 CaAsteroidIcy
+		}
+	}
+}
+
+@AsteroidSets[OPM_GasGiantObjects]:NEEDS[!UnhideAsteroids]
+{
+	@ASTEROIDGROUP:HAS[#centralBody[Jool]]
+	{
+		detectable
+		{
+			conditions
+			{
+				condition = Jool.reached
+			}
+		}
+	}
+	
+	@ASTEROIDGROUP:HAS[#centralBody[Sarnus]]
+	{
+		detectable
+		{
+			conditions
+			{
+				condition = Sarnus.reached
+			}
+		}
+	}
+	
+	@ASTEROIDGROUP:HAS[#centralBody[Urlum]]
+	{
+		detectable
+		{
+			conditions
+			{
+				condition = Urlum.reached
+			}
+		}
+	}
+	
+	@ASTEROIDGROUP:HAS[#centralBody[Neidon]]
+	{
+		detectable
+		{
+			conditions
+			{
+				condition = Neidon.reached
+			}
+		}
+	}
+}
+
+Localization
+{
+	en-us
+	{
+		// Minor satellites
+    		#autoLOC_CustomAsteroids_OPM_GroupInnerObj = Minor Sat.
+		// Captured Objects
+    		#autoLOC_CustomAsteroids_OPM_GroupOuterObj = Captured Ast.
+	}
+		fr-fr
+	{
+		// Minor satellites
+    		#autoLOC_CustomAsteroids_OPM_GroupInnerObj = Sat mineur.
+		// Captured Objects
+    		#autoLOC_CustomAsteroids_OPM_GroupOuterObj = Ast capturé.
```

The periods are part of the abbreviation; is the following correct French? What do the stock asteroid labels (in English, "Ast. XYZ-123") look like in the French version of the game?
```suggestion
		// Minor satellites
    		#autoLOC_CustomAsteroids_OPM_GroupInnerObj = Sat. mineur
		// Captured Objects
    		#autoLOC_CustomAsteroids_OPM_GroupOuterObj = Ast. capturé
```